test/mocha_integration_test.js

```diff
@@ -3117,4 +3117,73 @@ describe(
         expect(resBody.data.arrs.length).equal(1);
       });
 
-    })  
\ No newline at end of file
+    })  
+
+  describe('Helper connection for direct acces to nodes', function() {
+    //set up the environment
+    before(async function(){
+      //measurements for sql and zendro-server tests
+      let res = itHelpers.request_graph_ql_post('mutation{addBook(id:"remote_b1" title:"t1"){id} }');
+      expect(res.statusCode).to.equal(200);
+      res = itHelpers.request_graph_ql_post('mutation{addBook(id:"remote_b2" title:"t2"){id } }');
+      expect(res.statusCode).to.equal(200);
+      res = itHelpers.request_graph_ql_post('mutation{addAuthor(id:"remote_a1" name:"n1" addBooks:["remote_b1","remote_b2"]){id }}');
+      expect(res.statusCode).to.equal(200);
+      res = itHelpers.request_graph_ql_post('mutation{addAuthor(id:"remote_a2" name:"n2" addBooks:["remote_b1"]){id}}');
```

Comment:
add booksConnection for the test

Reply:
This is not the test, this is only the setup for the test, which is below.